zeebe-cluster/templates/configmap.yaml

```diff
@@ -25,6 +25,7 @@ data:
     
     exec /usr/local/zeebe/bin/broker
   zeebe.cfg.toml: |
+    {{ .Values.zeebeCfg }}
```

Comment:
@Zelldon this makes sense based on our discussions... I really don't like that kind of entire file injection.. but I am ok with this for now.